splunk_add_on_ucc_framework/commands/build.py

```diff
@@ -629,6 +635,15 @@ def generate(
         os.path.abspath(os.path.join(source, os.pardir, "additional_packaging.py"))
     ):
         sys.path.insert(0, os.path.abspath(os.path.join(source, os.pardir)))
+        try:
+            from additional_packaging import cleanup_output_files
+
+            cleanup_output_files(output_directory, ta_name)
+        except ImportError:
+            logger.info(
+                "additional_packaging.py is present but does not have `cleanup_output_files`. Skipping clean-up."
```

Comment:
If we decide to have 2 methods in the file and we create a file with only `cleanup_output_files`, an error will be thrown below:

```
            logger.exception(
                "additional_packaging.py is present but not importable.", e
            )
            raise e
```